app/controllers/component-tree.js

```diff
@@ -178,19 +178,15 @@ export default Controller.extend({
       });
     },
 
-    expandAll() {
+    /**
+     * Expand or collapse all component nodes
+     * @param {boolean} expanded If true, expanded, if false, collapsed
```

Comment:
`{Boolean}`

Reply:
Is that a standard to capitalize the types?